applications/smart-distancing/libs/loggers/csv_logger.py

```diff
@@ -1,130 +1,73 @@
 import csv
```

Comment:
Can you please create a new logger class file instead of completely rewriting this? I suspect we will need this later.

Reply:
Sure.